Source/Scene/Cesium3DTileset.js

```diff
@@ -1858,12 +1924,48 @@ define([
                 normal = Cartesian3.multiplyByScalar(normal, 0.75 * radius, scratchCartesian2);
                 position = Cartesian3.add(normal, boundingVolume.center, scratchCartesian2);
             }
-            tileset._geometricErrorLabels.add({
-                text: tile.geometricError.toString(),
-                position: position
+
+            var labelString = '';
+            var attributes = 0;
+
+            if (tileset.debugShowGeometricError) {
+                labelString += "\nGeometric error: " + tile.geometricError;
+                attributes++;
+            }
+            if (tileset.debugShowNumberOfCommands) {
+                labelString += "\nCommands: " + tile._commandsLength;
+                attributes++;
+            }
+            if (tileset.debugShowNumberOfPoints) {
+                labelString += "\nPoints: " + tile.content.pointsLength;
+                attributes++;
+            }
+            if (tileset.debugShowNumberOfTriangles) {
+                labelString += "\nTriangles: " + tile.content.trianglesLength;
+                attributes++;
+            }
+            if (tileset.debugShowNumberOfFeatures) {
+                labelString += "\nFeatures: " + tile.content.featuresLength;
+                attributes++;
+            }
+            if (tileset.debugShowTextureMemoryUsage) {
+                labelString += "\nTexture Memory: " + tile.content.textureMemorySizeInBytes;
+                attributes++;
+            }
+            if (tileset.debugShowVertexMemoryUsage) {
+                labelString += "\nVertex Memory: " + tile.content.vertexMemorySizeInBytes;
```

Comment:
Late suggestion, but can this and texture memory usage show it in megabytes like in the stats panel in the inspector?